x/examples/outline-connectivity-app/app_mobile/capacitor.config.ts

```diff
@@ -0,0 +1,27 @@
+import { CapacitorConfig } from '@capacitor/cli';
+
+let config: CapacitorConfig = {
+  appId: "org.getoutline.sdk.example.connectivity",
+  appName: "@outline_sdk_connectivity_demo/app_mobile",
+  webDir: "output/frontend",
+  server: {
+    url: "http://10.0.2.2:3000"
```

Comment:
Why does it need a local network address?

Reply:
Let's bind to localhost, so it doesn't risk external access. That's a security vulnerability for the developer (the dev app may have a vulnerability that allows for remote code execution, for example).